server/jetstream_cluster.go

```diff
@@ -1696,7 +1696,7 @@ func (js *jetStream) createRaftGroup(accName string, rg *raftGroup, storage Stor
 	}
 	js.mu.Unlock()
 
-	s.Debugf("JetStream cluster creating raft group:%+v", rg)
+	s.Debugf("JetStream cluster creating raft group:%+v for asset %q", rg, assetName)
```

Comment:
would be sweet if in the `StreamConfig` that is used to make the `S-` and `C-` streams we use the new metadata to also record the asset and account the raft asset relates to.

Reply:
2.10?

Reply:
right didnt see this targetted main, fair.

Reply:
Yes this will need to be for dev.